### PR TITLE
Fix for windows series.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-bundlelib/83/
Test PASSed.

File: jujubundlelib/references.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # The following regular expressions are the same used in juju-core: see
 # http://bazaar.launchpad.net/~go-bot/juju-core/trunk/view/head:/charm/url.go.
```
Comment (frankban) [inline jujubundlelib/references.py]: Could you please update this URL, from which these regexps are taken? We should really use the updated one on github.

File: jujubundlelib/references.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def from_jujucharms_url(cls, url):
         if match is None:
             msg = 'invalid charm or bundle URL: {}'.format(url)
             raise ValueError(msg.encode('utf-8'))
-        user, name, series, revision = match.groups()
+        user, name, series, drop, revision = match.groups()
```
Comment (frankban) [inline jujubundlelib/references.py]: s/drop/_/ ? [shrug]

**Comment by frankban** [comment]
:+1: with a couple of minor requests.
Thank you!

File: jujubundlelib/references.py (hunk 1 above)
Comment (fabricematrat) [inline jujubundlelib/references.py]: Done.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-bundlelib/84/
Test PASSed.

File: jujubundlelib/references.py (hunk 2 above)
Comment (fabricematrat) [inline jujubundlelib/references.py]: Done.

**Comment by jaycee** [comment]
:+1:

Does this need QA? And if so how does one go about QAing this?

**Comment by fabricematrat** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-bundlelib-merge

**Pull request merged**